netlab/L5/tls_utils.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <iostream>
+#include <array>
+#include <vector>
+#include <iterator>
+#include <string>
+#include <ctime>
+
+void append_size_to_string(std::string& str, std::size_t value) {
+	for (size_t i = 0; i < sizeof(std::size_t); ++i) {
+		str.push_back(static_cast<char>((value >> (i * 8)) & 0xFF));
+	}
+}
+
+/* Generates a random byte array. */
+template <std::size_t N>
+std::array<uint8_t, N> generate_random_bytes() {
```

Comment:
can we get a reference to array as argument and fill the array ?

Reply:
Sure, the retruend array is created inside, can be created outside and passed as argument. You want me to do it?

Reply:
Actually, I think it's better to leave it as is, works better with the way members are set in the constructors.